stream-enrich/0.17.0-rc1/google-pubsub/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base:0.1.0
```

Comment:
see #52

Reply:
Have any of them been changed?

Reply:
We just need to test (we really need to improve this dependency) this

Reply:
https://github.com/snowplow/snowplow-docker/pull/53/commits/6c48a032285745fc84440a675533d141bbd80b56

Reply:
@BenFradet 0.17.0 is out and this could be merged however this is not ready: https://github.com/snowplow/snowplow-docker/pull/53 As you know better the why and how of the base image change, can we merge this as is (removing the rc) and then you change the image on R6?